src/tablib/formats/_xlsx.py

```diff
@@ -3,6 +3,7 @@
 
 import re
 from io import BytesIO
+from typing import Optional, Union
```

Comment:
We only support Python 3.7 and up.

Can we remove this import and instead use `|` instead of `Union` and use `| None` instead of `Optional`?

We might need `from __future__ import annotations` instead.

Reply:
This doesn't work with Python 3.7:
```console
$ cat 1.py
def thing(bleep: str | None):
    ...

$ python3.7 1.py
Traceback (most recent call last):
  File "1.py", line 1, in <module>
    def thing(bleep: str | None):
TypeError: unsupported operand type(s) for |: 'type' and 'NoneType'
$ python3.7 -m mypy 1.py
1.py:1: error: X | Y syntax for unions requires Python 3.10
Found 1 error in 1 file (checked 1 source file)
```

But this does:
```console
$ cat 1.py
from __future__ import annotations

def thing(bleep: str | None):
    ...

$ python3.7 1.py
$ python3.7 -m mypy 1.py
Success: no issues found in 1 source file
```

But you can keep the `typing` imports if you prefer :)

Reply:
Thank you! I didn't get it for the first time. I've updated the typings with `from __future__ imports annotations` feature. Cound you check if this code is ready to be merged now?